### PR TITLE
:tada: Add new hero projects

### PR DESCRIPTION
**Pull request by EvaMarco**
This PR solves this US https://tree.taiga.io/project/penpot/us/3862?milestone=323955

### CODE REVIEW

File: frontend/src/app/main/ui/dashboard/projects.cljs [hunk 1]
```diff
@@ -51,10 +52,88 @@
...
+                       first)
+
+        on-template-cloned-success
+        (mf/use-callback
+         (fn [response]
+
+           (swap! state
+                  (fn [state]
+                    (-> state
+                        (assoc :status :success :file (:first response)))))
```
Comment (hirunatan) [inline frontend/src/app/main/ui/dashboard/projects.cljs]: perhaps this is better as an one liner
```clojure
(swap! state #(assoc % :status :success :file (:first response)))
```

(or in two lines, one with :status and other with :file)

File: frontend/src/app/main/ui/dashboard/projects.cljs [hunk 2]
```diff
@@ -51,10 +52,88 @@
...
+           (st/emit! (dd/go-to-workspace {:id (first response) :project-id default-project-id :name "tutorial"})
+                     (du/update-profile-props {:viewed-tutorial? true}))))
+
+        on-template-cloned-error
+        (fn []
+          (swap! state
+                 (fn [state]
+                   (-> state
+                       (assoc :status :waiting))))
```
Comment (hirunatan) [inline frontend/src/app/main/ui/dashboard/projects.cljs]: same here

File: frontend/src/app/main/ui/dashboard/projects.cljs [hunk 3]
```diff
@@ -51,10 +52,88 @@
...
+           (dm/error (tr "dashboard.libraries-and-templates.import-error"))))
+
+        download-tutorial
+        (fn []
+          (let [mdata  {:on-success on-template-cloned-success :on-error on-template-cloned-error}
+                params {:project-id default-project-id :template-id (:id template)}]
+            (swap! state
+                   (fn [state]
+                     (-> state
+                         (assoc :status :importing))))
```
Comment (hirunatan) [inline frontend/src/app/main/ui/dashboard/projects.cljs]: etc...

Comment (superalex) [reply frontend/src/app/main/ui/dashboard/projects.cljs]: Thanks! :)

File: frontend/src/app/main/ui/dashboard/projects.cljs (hunk 2 above)
Comment (superalex) [reply frontend/src/app/main/ui/dashboard/projects.cljs]: Thanks! :)

**Pull request merged**